lib/commands.py

```diff
@@ -240,6 +240,14 @@ def is_command(node):
     return CONNECTION_START in node.attrib or CONNECTION_END in node.attrib
 
 
+def is_command_symbol(node):
+    symbol = None
+    xlink = node.get(XLINK_HREF, "")
+    if len(xlink) > 11:
+        symbol = node.get(XLINK_HREF)[11:]
+    return symbol in OBJECT_COMMANDS or symbol in GLOBAL_COMMANDS
```

Comment:
Don't forget `LAYER_COMMANDS`. :)  `symbol in COMMANDS` should do the job.  `COMMANDS` should contain everything.